README.md

```diff
@@ -517,7 +517,7 @@ Language: English
 
 <img align="left" height="94px" width="94px" alt="Server Icon" src="images/server_icons/svelte.png" />
 
-[__svelte__](https://discord.gg/yy75DKs) [<img height="16px" width="16px" alt="Official Badge" src="images/badges/official.png">](badges.md#official-identification-badge) [<img height="16px" width="16px" alt="Homepage URL" src="images/badges/homepage.png">](https://svelte.dev/) [<img height="16px" width="16px" alt="Git Repository" src="images/badges/git.png">](https://github.com/sveltejs/svelte) \
+[__Svelte__](https://discord.gg/yy75DKs) [<img height="16px" width="16px" alt="Official Badge" src="images/badges/official.png">](badges.md#official-identification-badge) [<img height="16px" width="16px" alt="Homepage URL" src="images/badges/homepage.png">](https://svelte.dev/) [<img height="16px" width="16px" alt="Git Repository" src="images/badges/git.png">](https://github.com/sveltejs/svelte) \
 Notable Channels: `#svelte`, `#support`, `#sapper`, `#examples`, `#updates`, `#meetups`, `#future`, `#showcase`, `#nativedev`, `#internals`, `#gl`, **[`so much more`](badges.md#so-much-more)** \
```

Comment:
@ronCYA Can you replace the `**[so much more](badges.md#so-much-more)**` with ``#language-tools`, `#sveltesocietyday2020``? Since these two channels can fit in easily there's no need for [**`so-much-more`**](https://github.com/mhxion/awesome-programming-discord/blob/master/badges.md#so-much-more). You can put Svelte under [JavaScript Web Frameworks](https://github.com/mhxion/awesome-programming-discord#javascript-web-frameworks) or [Other JavaScript Libraries](https://github.com/mhxion/awesome-programming-discord#other-javascript-libraries) category for now, until Svelte gets more of its sibling discord communities where a distinct category for Svelte would really speak out.

I will update the guidelines later about that special tag and more about the categories 🙂

Reply:
Good point! There was also #jobs so I've added the three and removed 'so much more'.

Reply:
Looks good now. Thanks.